Assets/Scripts/ZoomSystem/DetailTextureImage.cs

```diff
@@ -0,0 +1,44 @@
+using UnityEngine;
+
+/**
+ * Controls the visibility of a texture depending on a Scanner's
+ * zoom level.
+ * TODO: Has a dependency on Scanner type
+ */
+[RequireComponent(typeof(Renderer))]
+public class DetailTextureImage : MonoBehaviour
+{
+    [SerializeField] Scanner scannedBy;
```

Comment:
I don't have a good idea of how you've planned the full thing. Do you think there will be a second Scanner? If not is it worth making it a singleton so that we don't have to add the reference every time?

Reply:
As per our conversation. Probably not a big deal. I wasn't comfortable with hardwiring a reference this early in the Scanner's development, and this is probably marginally easier to deal with if Scanner gets refactored or any of its listeners need tested in isolation.